Stripe/STPPaymentContext.m

```diff
@@ -510,9 +510,8 @@ - (void)appropriatelyDismissViewController:(UIViewController *)viewController
 - (BOOL)requestPaymentShouldPresentShippingViewController {
     BOOL shippingAddressRequired = self.configuration.requiredShippingAddressFields != STPBillingAddressFieldsNone;
     BOOL shippingAddressIncomplete = ![self.shippingAddress containsRequiredShippingAddressFields:self.configuration.requiredShippingAddressFields];
-    BOOL shippingMethodRequired = ([self.delegate respondsToSelector:@selector(paymentContext:didUpdateShippingAddress:completion:)] && !self.selectedShippingMethod);
     BOOL verificationRequired = self.configuration.verifyPrefilledShippingAddress && self.shippingAddressNeedsVerification;
-    return (shippingAddressRequired && (shippingAddressIncomplete || shippingMethodRequired || verificationRequired));
+    return (shippingAddressRequired && (shippingAddressIncomplete || verificationRequired));
```

Comment:
I think if they need the user to select a shipping method, turn off `verifyPrefilledShippingAddress` **and** pre-fill the shipping address, this will incorrectly skip selecting a shipping method.

Reply:
ah, good catch! updated to handle that case in `601d2f71d6`